core/src/main/java/com/datastrato/gravitino/storage/relational/utils/ExceptionUtils.java

```diff
@@ -14,12 +14,11 @@ private ExceptionUtils() {}
   public static void checkSQLConstraintException(
       RuntimeException re, Entity.EntityType type, String entityName) {
     if (re.getCause() != null
-        && re.getCause().getCause() != null
-        && re.getCause().getCause() instanceof SQLIntegrityConstraintViolationException) {
+        && re.getCause() instanceof SQLIntegrityConstraintViolationException) {
       // TODO We should make more fine-grained exception judgments
-      // Usually throwing `SQLIntegrityConstraintViolationException` means that
-      // SQL violates the constraints of `primary key` and `unique key`.
-      // We simply think that the entity already exists at this time.
+      //  Usually throwing `SQLIntegrityConstraintViolationException` means that
+      //  SQL violates the constraints of `primary key` and `unique key`.
+      //  We simply think that the entity already exists at this time.
```

Comment:
The changes here seems not necessary, am I right?

Reply:
After I removed the spaces locally, it could not highlight these lines of comments. I'm not sure if I need to make it highlighted.
![image](https://github.com/datastrato/gravitino/assets/26177232/f9e22111-65ab-435b-9739-1f38c0f9460a)
![image](https://github.com/datastrato/gravitino/assets/26177232/7a0f10d3-a84f-4e03-8c50-9ed632d57355)

Reply:
OK.